indigo_api/models/works.py

```diff
@@ -312,6 +313,27 @@ def friendly_type(self):
         if plugin:
             return plugin.work_friendly_type(self)
 
+    def possible_expression_dates(self):
```

Comment:
Docstring please -- in particular that it returns a list of dates in reverse order.